h/services/bulk_annotation.py

```diff
@@ -109,36 +109,34 @@ def annotation_search(
     @classmethod
     def _search_query(cls, authority, audience, updated) -> Select:
         """Generate a query which can then be executed to find annotations."""
-        query = sa.select(
-            [cls._AUTHOR.username, Group.authority_provided_id]
-        ).select_from(Annotation)
-
+        query = (
+            sa.select([cls._AUTHOR.username, Group.authority_provided_id]).select_from(
+                AnnotationSlim
+            )
+            # Always join on AUTHOR as we return its username
+            .join(cls._AUTHOR, cls._AUTHOR.id == AnnotationSlim.user_id)
+            # Always join on Group as we return the authority_provided_id
+            .join(Group, Group.id == AnnotationSlim.group_id)
+        )
         # Updated
-        query = query.where(date_match(Annotation.updated, updated))
+        query = query.where(date_match(AnnotationSlim.updated, updated))
```

Comment:
These just use the same column in the new table.

Reply:
:+1:

Reply:
Also already exists on `main` but I think you can do:

```python
query = (
    sa.select(...)
    .join(...)
    .join(...)
    .where(
        date_match(AnnotationSlim.updated, updated),
        AnnotationSlim.shared.is_(True),
        AnnotationSlim.deleted.is_(False),
        ...
    )
)
```

You don't need to do this thing where it goes:

```python
query = sa.select(...).join(...).join(...)

query = query.where(...)

query = query.where(...)

query = query.where(...)
```